datalad/support/s3.py

```diff
@@ -173,7 +172,8 @@ def set_bucket_public_access_policy(bucket):
       ]
     }""" % bucket.name)
 
-def gen_test_bucket(bucket_name):
+    # XXX: MIH: This syntax makes no sense!
+    def gen_test_bucket(bucket_name):
     conn = _get_bucket_connection(S3_ADMIN_CREDENTIAL)
```

Comment:
@yarikoptic Please have a look at this one. Seems to have been broken for quite some time.